pkg/apis/kibana/v1/kibana_types.go

```diff
@@ -98,16 +100,35 @@ func (k *Kibana) RequiresAssociation() bool {
 	return k.Spec.ElasticsearchRef.Name != ""
 }
 
-func (k *Kibana) AssociationStatus() commonv1.AssociationStatus {
-	return k.Status.AssociationStatus
+func (k *Kibana) AssociationStatusGroup(typ commonv1.AssociationType) commonv1.AssociationStatusMap {
+	if typ == commonv1.ElasticsearchAssociationType && k.Spec.ElasticsearchRef.IsDefined() {
+		return commonv1.NewAssociationStatusGroup(k.Spec.ElasticsearchRef.NamespacedName().String(), k.Status.AssociationStatus)
+	}
```

Comment:
I see these events (here on a Kibana association but others too) because we are not calling `WithDefaultNamespace` here and in the same method implementation for the other resources.

` Association status changed from [map[/elasticsearch:Established]] to [map[default/elasticsearch:Established]]`